src/__tests__/lib/containers/entity_finder/tree/TreeNode.test.tsx

```diff
@@ -239,6 +240,35 @@ describe('TreeNode tests', () => {
       'false',
     )
   })
+
+  it('Has aria-selected as true when the currentContainer matches the entity ID (BROWSE)', async () => {
```

Comment:
New test -- if currentContainer matches the node ID (which is typically synId or "root"), aria-selected should be true